modules/yieldmoBidAdapter.js

```diff
@@ -49,11 +49,10 @@ export const spec = {
             bidderRequest && bidderRequest.gdprConsent
               ? bidderRequest.gdprConsent.consentString
               : '',
-        })
-      ),
+        }),
       us_privacy:
```

Comment:
@MelodyLi2017 If `bidderRequest.uspConsent` is **undefined**, we get an empty query string parameter `us_privacy` with empty value.
<img width="1163" alt="Screenshot 2020-05-27 at 12 59 13 PM" src="https://user-images.githubusercontent.com/14229985/82990648-20e9ab80-a01a-11ea-81d3-24d1fd466440.png">

Is this intentional?

Reply:
yea! I think our ad server prefers empty string  vs. a back up constant vs. 'undefined'

Reply:
Okay, that sounds good.